src/mame/drivers/midyunit.cpp

```diff
@@ -3063,9 +3063,9 @@ ROM_END
  *
  *************************************/
 
-GAME( 1988, narc,       0,        zunit,                   narc,     midyunit_state, init_narc,     ROT0, "Williams", "Narc (rev 7.00)", MACHINE_SUPPORTS_SAVE )
-GAME( 1988, narc3,      narc,     zunit,                   narc,     midyunit_state, init_narc,     ROT0, "Williams", "Narc (rev 3.20)", MACHINE_SUPPORTS_SAVE )
-GAME( 1988, narc2,      narc,     zunit,                   narc,     midyunit_state, init_narc,     ROT0, "Williams", "Narc (rev 2.00)", MACHINE_SUPPORTS_SAVE )
+GAME( 1988, narc,       0,        zunit,                   narc,     midyunit_state, init_narc,     ROT0, "Williams", "NARC (rev 7.00)", MACHINE_SUPPORTS_SAVE )
+GAME( 1988, narc3,      narc,     zunit,                   narc,     midyunit_state, init_narc,     ROT0, "Williams", "NARC (rev 3.20)", MACHINE_SUPPORTS_SAVE )
+GAME( 1988, narc2,      narc,     zunit,                   narc,     midyunit_state, init_narc,     ROT0, "Williams", "NARC (rev 2.00)", MACHINE_SUPPORTS_SAVE )
```

Comment:
What are you basing this on?  As far as I can tell, it's an abbreviation for “narcotics”, not an initialism, and hence shouldn't be capitalised.  The Wikipedia page seems to agree with me: https://en.wikipedia.org/wiki/Narc_(video_game)

Reply:
OK. You can close this report. And i set up a new pull request with
Digger (Gremlin/Sega).